### PR TITLE
[new release] git, git-cohttp, git-cohttp-mirage, git-cohttp-unix, git-unix and git-mirage (3.1.0)

### PR DESCRIPTION
**Pull request by dinosaure**
Git format and protocol in pure OCaml

- Project page: <a href="https://github.com/mirage/ocaml-git">https://github.com/mirage/ocaml-git</a>
- Documentation: <a href="https://mirage.github.io/ocaml-git/">https://mirage.github.io/ocaml-git/</a>

##### CHANGES:

- Handle IP address on the endpoint (@dinosaure, mirage/ocaml-git#436)

### CODE REVIEW

**Comment by dinosaure** [comment]
Please, not merge today :+1:.

**Comment by dinosaure** [comment]
Ready for merge 👍 .

File: packages/git-unix/git-unix.3.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  "mirage-clock-unix"
+  "astring" {>= "0.8.5"}
+  "awa"
+  "cmdliner" {>= "1.0.4"}
+  "cohttp-lwt-unix" {>= "2.5.4"}
+  "decompress" {>= "1.2.0"}
+  "domain-name" {>= "0.3.0"}
+  "ipaddr" {>= "5.0.1"}
+  "mtime" {>= "1.2.0"}
+  "ocamlfind" {>= "1.8.1"}
```
Comment (avsm) [inline packages/git-unix/git-unix.3.1.0/opam:41]: Does `git-unix` really need ocamlfind?

Comment (dinosaure) [reply packages/git-unix/git-unix.3.1.0/opam:41]: I think it's needed for `git-unix.top` which is a simple REPL layer to load pretty-printers and libraries (`ocamlfind`) required.

Comment (dinosaure) [reply packages/git-unix/git-unix.3.1.0/opam:41]: I'm not very convince that this part is useful for the end-user but it's definitely good for the developer (who can introspect `git` with `dune buiild && ocaml`). If you think that we should remove this part, I can redo a release without it.

Comment (avsm) [reply packages/git-unix/git-unix.3.1.0/opam:41]: Not at all, I was just checking it hadn't snuck in by mistake. Thanks for the clarification :-)

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**